hs_core/hs_rdf.py

```diff
@@ -51,7 +51,7 @@ def ingest_metadata(self, graph):
         """Given an rdflib Graph, run ingest_rdf for all generic relations on the object"""
         subject = self.rdf_subject_from_graph(graph)
 
-        generic_relations = list(filter(lambda f: isinstance(f, GenericRelation), type(self)._meta.virtual_fields))
+        generic_relations = list(filter(lambda f: isinstance(f, GenericRelation), type(self)._meta.private_fields))
```

Comment:
can you explain this change? thanks...

Reply:
In django 1.1 the Options class had this `virtual_filelds` property (https://docs.djangoproject.com/en/1.11/_modules/django/db/models/options/#Options ) which is no more in 3.2:


```
def virtual_fields(self):
     return self.private_fields
```

Reply:
got it, thanks Pabitra, that makes sense